modules/ossm-kiali-ossmc-plugin-uninstall-cli.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-kiali-ossmc-plugin.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-kiali-ossmc-plugin-uninstall-cli_{context}"]
+= Uninstalling OpenShift Service Mesh Console plugin using the CLI
+
+You can uninstall the {SMProductName} (OSSMC) plugin by using the CLI.
+
+.Procedure
+
+. Remove the `OSSMC` CR by running the following command:
++
+[source,terminal]
+--
+ oc delete ossmconsoles <CR name> -n <CR namespace>
+--
++
+. Verify all CRs are deleted from all namespaces by running the following command:
++
+[source, terminal]
+--
+for r in $(oc get ossmconsoles --ignore-not-found=true --all-namespaces -o custom-columns=NS:.metadata.namespace,N:.metadata.name --no-headers | sed 's/  */:/g'); do oc delete ossmconsoles -n $(echo $r|cut -d: -f1) $(echo $r|cut -d: -f2); done
+--
```

Comment:
```suggestion
You can uninstall the {SMProductName} (OSSMC) plugin by using the {oc-first}.

.Procedure

. Remove the `OSSMC` custom resource (CR) by running the following command:
+
[source,terminal]
----
$ oc delete ossmconsoles <custom_resource_name> -n <custom_resource_namespace>
----

. Verify that all CRs are deleted from all namespaces by running the following command:
+
[source,terminal]
----
for r in $(oc get ossmconsoles --ignore-not-found=true --all-namespaces -o custom-columns=NS:.metadata.namespace,N:.metadata.name --no-headers | sed 's/  */:/g'); do oc delete ossmconsoles -n $(echo $r|cut -d: -f1) $(echo $r|cut -d: -f2); done
----
```

Pay attention to code block formatting per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output 🙏🏻